Description: change data path
 on system install, data path should be in /usr/share, so we have to change it in the code
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.c
+++ b/main.c
@@ -22,10 +22,10 @@
 	glLoadIdentity();
 
 	glOrtho(0.0,800.0,600.0,0.0,-10.0,600.0);
-	if ((movie = load_model("model.TVF")) == NULL) printf("Error!\n");
+	if ((movie = load_model("/usr/share/association-by-cyberneticgenetics/model.TVF")) == NULL) printf("Error!\n");
 
 	Mix_OpenAudio(22050, AUDIO_S16, 2, 4096);
-	music = Mix_LoadMUS("the.monotonic.ogg");
+	music = Mix_LoadMUS("/usr/share/association-by-cyberneticgenetics/the.monotonic.ogg");
 	Mix_PlayMusic(music, 0);
 
 	start = SDL_GetTicks();
